cypress/support/metaTestHelper.js

```diff
@@ -6,6 +6,18 @@ export const testResponseCode = (path, responseCode) => {
   });
 };
 
+export const retrieveCookieValue = (cookieName, value) => {
+  cy.getCookie(cookieName).should('have.property', 'value', value);
+};
+
+export const expectCookieExpiryDateOneYear = cookieName => {
```

Comment:
I'd prefer `expectDefaultCookieExpiry` (or maybe, `expectCookieExpiry(cookieName, DEFAULT_EXPIRY)`) to avoid putting a business rule that might change into the function name.